openadapt/temp_hierarchy_script/gpt_hierarchy.py

```diff
@@ -0,0 +1,55 @@
+import openai
+openai.api_key = "INSERT_API_KEY_HERE"
+
+completion = openai.ChatCompletion.create(
+    model="gpt-3.5-turbo",
+    messages=[
+        {"role":"system",
+            "content": "You are enumerating a hierarchy of actions that \
+                 one takes when operating GUI desktop applications for typical \
+                    day-to-day tasks. Consider different levels of abstractions. \
+                        Examples include: clicking a button, opening a window, \
+                            operating payroll software, generating invoices, \
+                                renting an apartment."},
```

Comment:
```

        {
            "role":"system",
            "content": (
                "You are enumerating a hierarchy of actions that "
                "one takes when operating GUI desktop applications for typical "
                "..."
             )
         }
```

Reply:
Even nicer:

prompt_lines = [
    "You are enumerating a hierarchy.",
    #"You are very smart.",
    (
        "This is a longer one "
        "spread over multiple lines."
    ),
    "Enumerate all different levels.",
]
prompt = "\n".join(prompt_lines)